src/fragments/lib/auth/js/mfa.mdx

```diff
@@ -203,6 +203,12 @@ try {
 </Block>
 </BlockSwitcher>
 
+<Callout warning>
+
+To avoid unexpected responses from Cognito, it is recommended to call the `Auth.setPreferredMFA()` API before retrieving MFA settings with `Auth.getPreferredMFA()`. If the MFA settings returned from Cognito do not align with the expected user’s preferences, trying to [setting the user MFA preferences](/lib/auth/mfa/q/platform/react-native/#setup-mfa-type) first.
```

Comment:
it is recommended to call the `Auth.setPreferredMFA()` when a user sets up their MFA method (or something similar).

Reply:
Updated the verbiage and condensed the callout.